astropy/wcs/wcsapi/high_level_api.py

```diff
@@ -43,8 +43,11 @@ def pixel_to_world(self, *pixel_arrays):
         Convert pixel coordinates to world coordinates (represented by high-level
         objects).
 
-        See `~astropy.wcs.wcsapi.BaseLowLevelWCS.pixel_to_world_values` for pixel indexing and
-        ordering conventions.
+        If a single high-level object is used to represent the world
+        coordinates, it is returned as-is (not in a tuple/list), otherwise a
```

Comment:
```suggestion
        coordinates (i.e., ``world_n_dim`` is 1), it is returned as-is (not in a tuple/list), otherwise a
```

Reply:
@eteq - actually this is not correct, hence the current wording. If you have a 2D WCS with celestial coordinates ``world_n_dim`` is 2 but we return a single high-level object.